mysql-test/suite/rocksdb/t/prefix_extractor_override.test

```diff
@@ -30,17 +30,12 @@ SELECT COUNT(*) FROM t1 WHERE id1=1 AND id2=1 AND id3=1;
 --sorted_result
 SELECT * FROM information_schema.rocksdb_cf_options WHERE option_type like '%prefix_extractor%';
 
-# This should no longer crash. See https://github.com/facebook/mysql-5.6/issues/641
+SET @@global.rocksdb_update_cf_options = 'cf1={prefix_extractor=capped:26};';
 --echo
---echo Restarting with new Prefix Extractor...
---echo
---let $restart_parameters=restart:--loose-rocksdb_override_cf_options=cf1={prefix_extractor=capped:26};
---source include/restart_mysqld.inc
-
---echo
---echo Changed Prefix Extractor (after restart):
+--echo Changed Prefix Extractor (after update_cf_options set, without restart):
 --echo
 --sorted_result
+# Restart no longer needed
```

Comment:
Perhaps the final restart at the end of testcase is no longer needed neither?

Reply:
Agreed, the removed restart was there to test through a crash that would have happened on server startup.  This final restart serves no purpose that I can see.